101-vm-with-standardssd-disk/azuredeploy.json

```diff
@@ -0,0 +1,249 @@
+            "description": "The name of the VM"
+          }
+      },
+      "adminUsername": {
+          "type": "string",
+          "defaultValue":"myadmin",
+          "metadata": {
+            "description": "The admin user name of the VM"
+          }
+      },
+      "adminPassword": {
+          "type": "securestring",
+          "metadata": {
+            "description": "The admin password of the VM"
+      }
+    },
+      "diskType": {
+          "type": "string",
+          "defaultValue": "StandardSSD_LRS",
+          "metadata": {
+            "description": "The Storage type of the data Disks"
+      }
+    },
+      "location": {
+          "type": "string",
+           "defaultValue":"northeurope",
+          "metadata": {
+            "description": "The Azure Region where VM is created"
+      }
+      },
+      "virtualMachineSize": {
+          "type": "string",
+           "defaultValue":"Standard_DS3_V2",
+           "metadata": {
+             "description": "The virtual machine size"
+       }
+      },
+      "virtualNetworkName": {
+          "type": "string",
+          "defaultValue":"myVnet",
+          "metadata": {
+            "description": "The virtual network where VM is hosted"
+      }
+      },
+      "subnetName": {
+          "defaultValue": "mySubnet",
+          "type": "String",
+          "metadata": {
+            "description": "The subnet where VM is hosted"
+      }
+      },
+      "windowsOSVersion": {
+          "type": "string",
+          "defaultValue": "2016-Datacenter",
+          "allowedValues": [
+              "2008-R2-SP1",
+              "2012-Datacenter",
+              "2012-R2-Datacenter",
+              "2016-Datacenter"
+          ],
+          "metadata": {
+              "description": "The Windows version for the VM. This will pick a fully patched image of this given Windows version. Allowed values: 2008-R2-SP1, 2012-Datacenter, 2012-R2-Datacenter, Windows-Server-Technical-Preview."
+          }
+      },
+      
+      "DataDisksCount": {
+          "defaultValue": 5,
+          "type": "int",
+          "metadata": {
+            "description": "The count of empty data disks created"
+      }
+      }
+      ,
+      
+      "DataDiskSize": {
+          "defaultValue": 4095,
+          "type": "int",
+          "metadata": {
+            "description": "The size of each data disk"
+      }
+      }
+      
+  },
+  "variables": {
+      "imagePublisher": "MicrosoftWindowsServer",
+      "imageOffer": "WindowsServer",
+      "location": "[resourceGroup().location]",
+      "OSDiskName": "[concat(toLower(parameters('virtualMachineName')),'OSDisk')]",
+      "addressPrefix": "10.2.3.0/24",
+      "subnetPrefix": "10.2.3.0/24",
+      "publicIPAddressType": "Dynamic",
+      "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',parameters('virtualNetworkName'))]",
+      "subnetRef": "[concat(variables('vnetID'),'/subnets/', parameters('subnetName'))]",
```

Comment:
use:
[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('virtualNetworkName'), variables('subnetName'))]

And remove vnetID above as it isn't needed